policy/enforcer.go

```diff
@@ -31,6 +31,27 @@ type Enforcer struct {
 	PolicyLoader Loader
 }
 
+// type for Identity Token claims
+type claims struct {
+	Email  string   `json:"email"`
+	Sub    string   `json:"sub"`
+	Groups []string `json:"groups"`
+}
+
+// Validates that the server defined identity attribute matches the
+// respective claim from the identity token
+func validateClaim(claims *claims, user *User) bool {
+	if strings.HasPrefix(user.EmailOrSub, "oidc:groups") {
```

Comment:
I agree with you, we should definitely rename EmailOrSub as it is a confusing name with this new feature.

Reply:
Updated the name, references and associated documentation.